app/views/users/posts/new.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1>New post</h1>
+
+<%= form_with(model: [@user, @post], url: user_posts_path(user_id: @user)) do |f| %>
+  <div>
+    <%= f.label :title %> <br>
+    <%= f.text_field :title %>
+  </div>
+  <div>
+    <%= f.label :text %> <br>
+    <%= f.text_area :text %>
+  </div>
+  <%= f.submit %>
+<% end %>
```

Comment:
- [ ] When I attempt to create a new post after clicking create post in the form, the app produces an error 👇🏼 . Kindly make sure to fix this so your app works as expected and the user can create a new post

![Screenshot_20230513_212306](https://github.com/kensteph/blog/assets/25227022/15be2918-1562-408b-bf21-ba460e3507ec)